### PR TITLE
show_bottom_toolbar - Feature to Show/Hide Toolbar

### PR DESCRIPTION
**Pull request by stephanoparaskeva**
First open source commit! 🍻

## Description
This change introduces `show_bottom_toolbar` config option in pgclirc of True or False. Defaulting to True.

### If False the toolbar will be hidden.
Screenshot 2020-04-17 at 23 34 34

### If True the toolbar will be visible.
Screenshot 2020-04-17 at 23 35 05

This change was requested [here](https://github.com/dbcli/pgcli/issues/939#issuecomment-615469759).

## Checklist

- [x] I've added this contribution to the `changelog.rst`.
- [x] I've added my name to the `AUTHORS` file (or it's already there).

- [x] I installed pre-commit hooks (`pip install pre-commit && pre-commit install`), and ran `black` on my code.
- [x] Please squash merge this pull request (uncheck if you'd like us to merge as multiple commits)

### CODE REVIEW

File: pgcli/main.py [hunk 1]
```diff
@@ -160,6 +160,7 @@ def __init__(
         prompt_dsn=None,
         auto_vertical_output=False,
         warn=None,
+        show_bottom_toolbar=False,
```
Comment (j-bennet) [inline pgcli/main.py:163]: You added a parameter here, but it's not being used. The parameter would be needed if you also had a command line option to disable or enable the toolbar on-the-fly, i.e by running `pgcli --no-toolbar` (example option [here](https://github.com/dbcli/pgcli/blob/f3ac5598448a0c2080ee267f00931ed556a6779d/pgcli/main.py#L1113-L1120)), but I think that would be an overkill. Config option is enough.

Comment (stephanoparaskeva) [reply pgcli/main.py]: I see, need to turn on pylint, and I could try to work on that next time if I get some time. In the meantime, I have removed the unused var as per your comment.

**Comment by stephanoparaskeva** [comment]
> Congratulations on your first open-source PR. :) Well done. Please add your name and the change to changelog also:
> 
> https://github.com/dbcli/pgcli/blob/master/changelog.rst

Cheers! Made requested change to changelog.

**Comment by stephanoparaskeva** [comment]
Also If I want to remove the \n newline for my prompt locally, where is this located? I want to just disable newline that is inserted with normal prompt, personally (not for pr)

**Comment by j-bennet** [comment]
> Also If I want to remove the \n newline for my prompt locally, where is this located?

Not sure about that. @jonathanslenders can you advise?

**Comment by j-bennet** [comment]
@stephanoparaskeva Nice work - thank you! 🍓

**Comment by laixintao** [comment]
> > Also If I want to remove the \n newline for my prompt locally, where is this located?
> 
> Not sure about that. @jonathanslenders can you advise?

This will fix the empty newline issue. https://github.com/dbcli/pgcli/pull/1171

But I am not sure if we want this. @j-bennet

**Pull request merged**